Flaky test_generic_driver.py::test_kill_gives_correct_state[LsfDriver]

Observed in https://github.com/equinor/komodo-releases/actions/runs/16017830600/job/45188642503

`
FAILED unit_tests/scheduler/test_generic_driver.py::test_kill_gives_correct_state[LsfDriver] - assert 15 in [143, 130, 265, 271]`

This is when tested onprem with the real LSF cluster (the `--lsf` option to pytest) at work:
`pytest --timeout=3600 -v --lsf --basetemp=/private/f_scout_ci/pytest-tmp/tmp.VMvD4mO9Hp unit_tests/scheduler`

--- a/tests/ert/unit_tests/scheduler/test_generic_driver.py
+++ b/tests/ert/unit_tests/scheduler/test_generic_driver.py
@@ -101,8 +101,9 @@ async def test_kill_gives_correct_state(driver: Driver, use_tmpdir, request):
         ]
     else:
         expected_returncodes = [
-            SIGNAL_OFFSET + signal.SIGTERM,
+            signal.SIGTERM,
             SIGNAL_OFFSET + signal.SIGINT,
+            SIGNAL_OFFSET + signal.SIGTERM,
             256 + signal.SIGKILL,
             256 + signal.SIGTERM,
         ]

